README.md

```diff
@@ -3,7 +3,7 @@ HttpSysServer
 
 | AppVeyor | Travis |
 | ---- | ----
-| [![AppVeyor](https://ci.appveyor.com/api/projects/status/413hlt87c3p24on8/branch/dev?svg=true)](https://ci.appveyor.com/project/aspnetci/WebListener/branch/dev) | [![Travis](https://travis-ci.org/aspnet/WebListener.svg?branch=dev)](https://travis-ci.org/aspnet/WebListener) |
```

Comment:
Not as simple as it seems for AppVeyor! Send me an email as a reminder and I'll get it done.